### DISCUSSION TITLE
Set up Magento with the automated onelinesetup

### DISCUSSION THREAD

**Discussion by firasmf** [markshust/docker-magento#618] [General]
what is the issue and how I can fix it , please ?!!!

 [image link](https://cdn.filestackcontent.com/9MQj0dvnRNmAzkXtXLI5)

**Comment by YevhenZvieriev** [comment]
Hi, @firasmf 

Check this [link](https://github.com/markshust/docker-magento#install-fails-because-project-directory-is-not-empty)
